configs/s2ef/200k/schnet/schnet.yml

```diff
@@ -17,10 +17,10 @@ optim:
   lr_initial: 0.0005
   lr_gamma: 0.1
   lr_milestones: # epochs at which lr_initial <- lr_initial * lr_gamma
-    - 5
-    - 8
-    - 10
-  warmup_epochs: 3
+    - 4000
```

Comment:
I'm guessing that this would break backward compatibility with old configs, right? 
Do we want to support both styles in order to be backward compatible?

Reply:
Agreed - either that or we update the old configs with the appropriate steps those epochs corresponded to.

Reply:
I'll update all the old configs. It's easier to do that once instead of having additional code for these two cases.